vignettes/dm.Rmd

```diff
@@ -232,6 +232,25 @@ Unfortunately, this currently only works for MSSQL and Postgres, so we cannot sh
 flights_dm_from_remote <- dm_learn_from_db(src_sqlite)
 ```
 
+## Adding tables and retrieving results
```

Comment:
These changes belong in a different pull request?

Please branch off of master for the next pull request. Don't worry for this pull request, I'll fix.